airbyte-webapp/src/packages/cloud/lib/auth/GoogleAuthService.ts

```diff
@@ -89,6 +90,13 @@ export class GoogleAuthService implements AuthService {
     );
   }
 
+  async updateProfile(displayName: string): Promise<void> {
```

Comment:
Do we still need this method, or can we remove it if unused?

Reply:
It's not used anymore, but let's keep it as it was already added.